src/category/basic.lean

```diff
@@ -95,6 +97,24 @@ by rw [← bind_pure_comp_eq_map, bind_assoc]; simp [pure_bind]
 lemma seq_eq_bind_map {x : m α} {f : m (α → β)} : f <*> x = (f >>= (<$> x)) :=
 (bind_map_eq_seq m f x).symm
 
+/-- This is the Kleisli composition -/
+@[reducible] def pipe {m} [monad m] {α β γ} (f : α → m β) (g : β → m γ) := λ x, f x >>= g
+
+infix ` >=> `:55 := pipe
```

Comment:
I think I asked about this earlier, but `>=>` is already defined as an infix operator in core Lean (`category.combinators`) with, as far as I can tell, pretty much the same definition. If there is a purpose to the definition here, won't that be potentially confusing?

Reply:
Here is the definition in question: https://github.com/leanprover/lean/blob/master/library/init/category/combinators.lean#L22

The difference is significant: the precedence here is 55, comparable to that of `>>=` and I use a named definition instead of a lambda. I have found the version of core to be unusable and I know of no example where it is used.

Reply:
Perhaps it's worth adding a note here, at the new notation definition, explaining that the core version is unusable. (Just so that someone eventually removes it, when that becomes possible.)

Reply:
Okay, I guess it probably won't cause any problems. But maybe it would still be a good idea to add a comment acknowledging the fact that `>=>` is already defined in the core library but it is unusable for [insert reason here].

Reply:
done